### PR TITLE
Fix background colour on overscroll

### PR DESCRIPTION
**Pull request by NoahvdAa**
When "overscrolling", the background of the page shows as white. This pr fixes that.

before:
https://user-images.githubusercontent.com/44026893/131344440-f00ccfbd-32d0-4393-8474-a1198c2bf9dd.mov
after:
https://user-images.githubusercontent.com/44026893/131344650-3242e756-a5a6-427e-8a3a-864e0f19074a.mov

### CODE REVIEW

File: frontend/assets/theme/base.scss [hunk 1]
```diff
@@ -14,3 +14,6 @@
 a {
   text-decoration: none;
 }
+body {
+  background-color: #272727 !important;
+}
```
Comment (Machine-Maker) [inline frontend/assets/theme/base.scss:19]: Add newline at the bottom. I also swear theres a variable for the `#272727` color? If there is, should use that.

**Review by Machine-Maker — CHANGES_REQUESTED** [review]

File: frontend/assets/theme/base.scss (hunk 1 above)
Comment (NoahvdAa) [reply frontend/assets/theme/base.scss]: I couldn't find a variable, the colour only seems to be in the `.v-card, .v-sheet.v-list` css block.

**Review by Machine-Maker — APPROVED** [review]

**Comment by MiniDigger** [comment]
thanks

**Comment by MiniDigger** [comment]
oops, wrong button

**Pull request merged**